File: day_four.py

user_input = int(input("Enter the month of the year (from 1 - 12): "))
if user_input >= 1 and user_input <= 3:
    print("First quarter of the year")
elif user_input >= 4 and user_input <= 6:
    print("Second quarter of the year")
elif user_input >= 7 and user_input <= 9:
    print("Third quarter of the year")
elif user_input >= 10 and user_input <= 12:
    print("Fourth quarter of the year")
else:
    print("Invalid user input")

File: DayFour.java
import java.util.Scanner;

public class DayFour{
	public static void main(String[] args){
		Scanner inputCollector = new Scanner(System.in);

		System.out.print("Enter the month of the year (from 1 - 12): ");
		int userInput = inputCollector.nextInt();
		
		if ( userInput >= 1 && userInput <= 3){
			System.out.print("First quarter of the year");
		}
		else if ( userInput >= 4 && userInput <= 6){
			System.out.print("Second quarter of the year");
		}
		else if ( userInput >= 7 && userInput <= 9){
			System.out.print("Third quarter of the year");
		}
		else if ( userInput >= 10 && userInput <= 12){
			System.out.print("Fourth quarter of the year");
		}
		else {
			System.out.print("Invalid input");
		}
		
	}
}